torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -789,12 +789,6 @@ at::Tensor AtenXlaType::bmm(const at::Tensor& self, const at::Tensor& mat2) {
       XLATensor::bmm(bridge::GetXlaTensor(self), bridge::GetXlaTensor(mat2)));
 }
 
-std::vector<at::Tensor> AtenXlaType::broadcast_tensors(at::TensorList tensors) {
-  XLA_FN_COUNTER("xla::");
-  return bridge::AtenFromXlaTensors(
-      XLATensor::broadcast_tensors(bridge::GetXlaTensors(tensors)));
```

Comment:
Is the XLATensor API still used?

Reply:
Yea it's still used by `SmoothL1Loss` and its backward.